smartsim/_core/generation/generator.py

```diff
@@ -240,57 +233,50 @@ def _symlink_files(files: t.Union[EntityFiles, None], dest: pathlib.Path) -> Non
     @staticmethod
     def _write_tagged_files(
         files: t.Union[EntityFiles, None],
-        params: t.Mapping[str, str],
+        params: t.Mapping[str, t.Any],
```

Comment:
Any reason for the type change here? I think it should be a okay to keep this as a `Mapping[str, str]`. It doesn't make much sense to write anything but a string into a text file imo